libraries/aws/provisioning/src/aws_iot_provisioning_api.c

```diff
@@ -230,6 +236,17 @@ static void _keysAndCertificateResponseReceivedCallback( void * param1,
 
 /*-----------------------------------------------------------*/
 
+static void _csrResponseReceivedCallback( void * param1,
+                                          IotMqttCallbackParam_t * const pPublish )
+{
+    /* Silence warnings about unused variables.*/
+    ( void ) param1;
+
+    _commonServerResponseHandler( pPublish, _AwsIotProvisioning_ParseCsrResponse );
```

Comment:
Why not pass param1 all the way through to the _common handler?

Reply:
As it is not required by the `_commonServerResponseHandler()` function. I would have to use the same `(void) param1` logic in the common function.